### ISSUE TITLE
Paladins Cannot Use Shield of Righteousness when Silenced

### ISSUE THREAD

**Issue by Skipslegdays** [dalaranwow/dalaran-wow#5959]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

When silenced, Shield of Righteousness is not usable

**Expected behaviour**: Tell us what should happen instead

It should be usable since it is a weapon strike ability

**Steps to reproduce the problem**:

1.  Be a paladin with a shield equipped
2.  Get silenced
3.  Try to use shield of righteousness

**Include proofs for this behaviour**

"Hammer of the Righteous and Shield of Righteousness are both not affected by silences, those two alone should keep you going through a silence effect as they are two of your highest threat abilities"
From a post back in 2009
https://www.mmo-champion.com/threads/650747-Spell-tanking-while-silenced

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
